src/css/scss/chatbot-widget.scss

```diff
@@ -208,14 +221,25 @@ $text-color-xlight: $white;
      */
 
     &__text,
+    &__image,
     &__url {
         background-color: $grey-lighter;
-        padding: $spacer-xs $spacer-sm;
+        border-radius: $border-radius;
+        border-bottom-left-radius: 0;
+        padding: $spacer-xs / 6 * 5 $spacer-sm / 9 * 7; // 10px 14px
+    }
+
+    &__image {
+        padding: $spacer-sm / 9 * 7;
+
+        img {
+            max-width: 100%;
+        }
     }
 
     &__error {
         background-color: mix($danger, $white, 10%);
-        padding: $spacer-xs $spacer-sm;
+        padding: $spacer-xs / 6 * 5 $spacer-sm / 9 * 7; // 10px 14px => min-height = 36px
```

Comment:
readability would be improved with ()

Reply:
True, but in the last-known-to-me code guidelines it's prohibited 😇